frontend/src/components/pageNotFound/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Grid, Box } from '@material-ui/core';
+import CancelIcon from '@material-ui/icons/CancelRounded';
+import { useLocation } from 'react-router-dom';
+import errorPageStyle from './pageNotFound.module.scss';
+
+const PageNotFound = () => {
+  const location = useLocation();
+  return (
+    <div className={errorPageStyle.errorPage}>
+      <Grid
+        container
+        alignItems="center"
+        justify="center"
+        style={{ minHeight: '100vh' }}>
+        <Box>
```

Comment:
I'm not very familiar with material UI, do we need both Grid and Box here?
If it's used for the alignment, can we just use Box? 
Can we use div and move the styling to css file (with `display: flex` etc.)?

Reply:
I've had a play around and nope I don't think we do. I was following some stack overflow advice on Material UI components. I've also not used it before - it's slightly different from Semantic UI which I have used before.